docs/cep/sink/sink-types/stream-sink.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Stream Sink
+---
+
+Stream sinks consume events from streams and publish them using multiple transports to external endpoints in various data formats.
+
+## Purpose
+
+Stream sink provides a way to publish stream events of a stream to external systems by converting events to their supported format.
+
+## Syntax
+
+To configure a stream to publish events via a sink, add the sink configuration to a stream definition by adding the `sink.type` annotation with the required parameter values.
+
+The sink syntax is as follows:
+
+```sql
+   CREATE SINK SinkName WITH (type="stream", stream="STRING", replication.type="STRING", map.type='type') (strings);
+```
+
+## Query Parameters
```

Comment:
@StoyanTc - This list of query parameters does not seem to match the syntax shown above. Is it correct?

Reply:
@diana-macrometa - You're right. Only the `stream source` has `stream.list`.

Reply:
So how should I change it? If you want, you can make the change as a suggestion or a commit.

Reply:
Actually, I think I might have found the right one.

Reply:
Yes, that's correct. You was faster than me.

Reply:
You gave me the clue, I copy/pasted the wrong table. So based on what you said, I was able to find the right table.